Misc/NEWS.d/next/Library/2020-10-15-18-37-12.bpo-42047.XDdoSF.rst

```diff
@@ -0,0 +1 @@
+Add :func:`threading.get_native_id` support for DragonFly BSD.
```

Comment:
Feel free to also credit yourself in the NEWS entry.

Reply:
Jesus I almost forgot about this one, thanks for the changes.